packages/tools/src/tools/displayTools/Labelmap/labelmapDisplay.ts

```diff
@@ -166,6 +200,9 @@ async function render(
     throw new Error(`No Labelmap found for volumeId: ${labelmapUID}`);
   }
 
+  if (!isSameFrameOfReference(viewport, labelmapData?.referencedVolumeId)) {
```

Comment:
is this correct? should it be referencedVolumeId or referenceSeriesInstanceUID?

Reply:
It is faster if use the referencedVolumeId, because I need the FrameOfReference of the referencedSeries

Reply:
then why did you add the referenceSeriesInstanceUID at all? I don't see that being used anywhere